releng/source-tarball/fs/usr/local/bin/influxdb_tarball.bash

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+function printHelp() {
+  >&2 echo "USAGE: $0 -s INFLUXDB_SHA -b INFLUXDB_BRANCH -v INFLUXDB_VERSION
+
+Emits a tarball of influxdb source code and dependencies to /out,
+which must be a mounted volume if you want to access the file.
+
+If the directory /influxdb-git exists and is mounted,
+that will be used as the git repository used when cloning plutonium.
+"
+}
+
+if [ $# -eq 0 ]; then
+  printHelp
+  exit 1
+fi
+
+SHA=""
+BRANCH=""
+VERSION=""
+
+while getopts hs:b:v: arg; do
+  case "$arg" in
+    h) printHelp; exit 1;;
+    s) SHA="$OPTARG";;
+    b) BRANCH="$OPTARG";;
+    v) VERSION="$OPTARG";;
+  esac
+done
+
+if [ -z "$SHA" ] || [ -z "$BRANCH" ] || [ -z "$VERSION" ]; then
+  printHelp
+  exit 1
+fi
+
+IPATH=/go/src/github.com/influxdata
+mkdir -p "$IPATH" && cd "$IPATH"
+if [ -d /influxdb-git ]; then
+  git clone /influxdb-git "$IPATH/influxdb"
+else
+  # Add GitHub host key to known hosts, non-interactively.
+  ssh-keyscan github.com >> ~/.ssh/known_hosts
+
+  git clone git@github.com:influxdata/influxdb.git
```

Comment:
Since this is a public repo, we can clone over HTTPS and skip the `ssh-keyscan`.